src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = ['Under construction'];
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: () => 'Under construction',
+  },
+});
```

Comment:
- [ ] Great work so far! However, according to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions_v2.md), the reducer inside the categories slice needs to check the status before updating the state. It appears that this step has been missed in your code, so please ensure that you check the payload from the action before making any updates to the state, Here is an example of how to do this:
```jsx
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories

```
- [ ] As you might've guessed this also means your initial state needs to be an object with a property called `categories` that holds an array as it's value. 😉